Bottom: 941649396eb9e3d935ba7ded567be7edaa265305
Top:    338bbd4e7a43c4875f6f373cb2eb623c3e6c6fdc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 19:52:07 +0000

Refresh of remove-unused-hid-method-inval

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 2a92514..2fec418 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3766,7 +3766,6 @@ HID lesstif_gui = {
   lesstif_get_export_options,
   lesstif_do_export,
   lesstif_parse_arguments,
-
   lesstif_invalidate_lr,
   lesstif_invalidate_all,
   lesstif_set_layer,
